lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -457,4 +457,9 @@ private function parseColumnCommentFromSQL($column, $sql)
 
         return false;
     }
+
+    private function escapeRegex($str)
```

Comment:
I think this is the wrong solution, while the bug spawns from no parameters being passed to `preg_quote()`. We should probably just switch the delimiter to be `/` (as usual) and then it should be fine. Can you give that a spin?

Reply:
Sure, I'll test it..